examples/WORKSPACE

```diff
@@ -49,3 +49,5 @@ examples_third_party_repositories()
 load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
 
 rbe_autoconfig(name = "buildkite_config")
+
+register_toolchains("@rules_foreign_cc_examples_third_party//python:python_toolchain")
```

Comment:
Can you move this into `./examples/third_party/python/python_repositories.bzl`?

Reply:
I tried that but `buildifier` complained about the fact that the functions used in `examples/third_party/repositories.bzl` do not have a `name` argument. This is because `buildifier` considers functions as "macros" if `native` is being used, and when `register_toolchains` was moved to `python_respositories.bzl` I had to invoke it as `native.register_toolchains`.

Reply:
If you move that into a macro, you need to use the `native` module, and as for the buildifier isssue, you can annotate that away:
https://github.com/bazelbuild/rules_foreign_cc/blob/98c88de5e0f57dd2a896dd595469ccd6bf97ceab/foreign_cc/repositories.bzl#L11-L12

That should fix the issue and keep the functionality for the python tests primarily in one place.

Reply:
done